server/src/app.ts

```diff
@@ -14,10 +14,24 @@ import {SubscriptionServer} from "subscriptions-transport-ws";
 import {createServer} from "http";
 import {findUserByID} from "./database";
 import flash = require("connect-flash");
+const fetch = require('isomorphic-fetch')
+const bodyParser = require('body-parser')
 
 // Set up Express and its middleware
 export let app = express();
 
+app.post("/api/slack/feedback", bodyParser.json(), (req, res) => {
+  fetch("https://hooks.slack.com/services/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", {
+    method: "POST",
+    body: JSON.stringify(req.body)
+  }).then(response => {
+    return res.status(response.status).send({
+      status: response.status,
+      statusText: response.statusText
+    });
+  });
+});
+
```

Comment:
This is essentially the correct way to define this, but a few issues:
1. This code should be inside of the API routes file (see line 93 of this file https://github.com/HackGT/bolt/blob/bf69496dd6e75650da6b1499b1c62ead74479b2e/server/src/app.ts#L93).  The API routes file is in src/api/api.ts and the routes are defined at the very bottom.
2. Add a `.catch` on the promise chain in the event that the POST request fails.  When the request fails you should log the exact error and return an API response indicating that an error occurred
2. Use env vars for the URL (which you know)